rb/lib/selenium/webdriver/firefox/profiles_ini.rb

```diff
@@ -57,11 +57,10 @@ def parse
               is_relative = Regexp.last_match(1).strip == '1'
             when /^Path=(.+)$/
               path = Regexp.last_match(1).strip
+              p = path_for(name, is_relative, path)
+              @profile_paths[name] = p if p
```

Comment:
Moving this code inside the last case statement doesn't seem right. Why are we defining `name` and `is_relative` variables there if we aren't using them anywhere?